pkg/kubectl/cmd/annotate.go

```diff
@@ -107,7 +109,7 @@ func NewCmdAnnotate(f cmdutil.Factory, out io.Writer) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:     "annotate [--overwrite] (-f FILENAME | TYPE NAME) KEY_1=VAL_1 ... KEY_N=VAL_N [--resource-version=version]",
-		Short:   "Update the annotations on a resource",
+		Short:   T("annotate_short"),
```

Comment:
> And even better, you don't even need the generated file because if a messages file for the wanted language is not present, you just default for what T takes as an argument... messages file generation can be used only as a stub for translating the other languages.

Don't we need stable keys to be able to locate the translated string?  That would seem to require putting some identifier in the code to keep the key stable for file moves and the like.  It's also not obvious how this approach would deal with, "I want the same message as that guy over there".